projects/Mallard/android/app/src/main/java/com/guardian/editions/MainApplication.java

```diff
@@ -34,7 +35,8 @@ public boolean getUseDeveloperSupport() {
 
             packages.add(new ReleaseStreamPackage());
 
-            // packages.add(new MainReactPackage());
+            // packages.add(new MainReactPackage(),
+            packages.add(new RNSentryPackage());
```

Comment:
I think that's no supposed to needed with RN's auto-linking?

Reply:
Sentry is not up to date with the latest strategies.